datalad/support/sshconnector.py

```diff
@@ -97,6 +98,9 @@ def get_connection_hash(hostname, port='', username='', identity_file='',
 class BaseSSHConnection(object):
     """Representation of an SSH connection.
     """
+
+    env_name_datalad_windows_ssh_client = "DATALAD_WINDOWS_SSH_CLIENT"
```

Comment:
Is there a reason to not use the standard config mechanism to define `datalad.windows.ssh-client` instead of limiting it to env var?

Reply:
No, I am just not used to datalad's standard config mechanism ;-), and there were a few other places that used env-vars.

Will change it to config. Good idea!